test/e2e/deployment.go

```diff
@@ -270,3 +274,131 @@ func testRollingUpdateDeploymentEvents(f *Framework) {
 	Expect(events.Items[0].Message).Should(Equal(fmt.Sprintf("Scaled up rc %s to 1", newRC.Name)))
 	Expect(events.Items[1].Message).Should(Equal(fmt.Sprintf("Scaled down rc %s to 0", rcName)))
 }
+
+func testRolloverDeployment(f *Framework) {
```

Comment:
Would be good to add a comment here describing briefly here what this function tests.